src/features/offer/components/ApplicationProcessingModal/ApplicationProcessingModal.native.test.tsx

```diff
@@ -30,4 +33,40 @@ describe('<ApplicationProcessingModal />', () => {
       expect(navigate).toHaveBeenCalledWith('TabNavigator', { screen: 'Profile' })
     })
   })
+
+  it('should log analytics when clicking on button "Aller sur mon profil', async () => {
+    const { getByLabelText } = render(
+      <ApplicationProcessingModal visible hideModal={hideModal} offerId={offerId} />
+    )
+
+    fireEvent.press(getByLabelText('Aller sur mon profil'))
+
+    expect(analytics.logGoToProfil).toHaveBeenCalledWith({
+      from: 'ApplicationProcessingModal',
+      offerId,
+    })
+  })
+
+  it('should close modal when clicking on button "Mettre en favori', async () => {
+    const { getByText } = render(
+      <ApplicationProcessingModal visible hideModal={hideModal} offerId={offerId} />
+    )
+
+    fireEvent.press(getByText('Mettre en favori'))
+
+    expect(hideModal).toBeCalledTimes(1)
```

Comment:
Pour rester cohérent
```suggestion
    expect(hideModal).toHaveBeenCalledTimes(1)
```